agent/acs/handler/payload_handler_test.go

```diff
@@ -754,3 +755,31 @@ func TestPayloadHandlerAddedASMAuthData(t *testing.T) {
 	assert.Equal(t, aws.StringValue(expected.AsmAuthData.Region), actual.ASMAuthData.Region)
 	assert.Equal(t, aws.StringValue(expected.AsmAuthData.CredentialsParameter), actual.ASMAuthData.CredentialsParameter)
 }
+
+func TestHandleUnrecognizedTask(t *testing.T) {
+	tester := setup(t)
+	defer tester.ctrl.Finish()
+
+	arn := "arn"
+	ecsacsTask := &ecsacs.Task{Arn: &arn}
+	payloadMessage := &ecsacs.PayloadMessage{
+		Tasks: []*ecsacs.Task{ecsacsTask},
+		MessageId: aws.String(payloadMessageId),
+	}
+
+	mockECSACSClient := mock_api.NewMockECSClient(tester.ctrl)
+	taskHandler := eventhandler.NewTaskHandler(tester.ctx, tester.payloadHandler.saver, nil, mockECSACSClient)
+	tester.payloadHandler.taskHandler = taskHandler
+
+	wait := &sync.WaitGroup{}
+	wait.Add(1)
+
+	mockECSACSClient.EXPECT().SubmitTaskStateChange(gomock.Any()).Do(func(change api.TaskStateChange) {
+		assert.NotNil(t, change.Task)
```

Comment:
Do we allow unrecognized tasks to run and complete? If so, how do we know that the task state changes after this are handled ok? 

i.e. Is the empty task passed on to another component? And if so, we we creating a new untested code path by creating the empty task?